Domain/Music_Portal.Domain.Core/Artist.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Core
+﻿namespace Domain.Core
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
namespace Music_Portal.Domain.Core